lib/properties.ex

```diff
@@ -92,9 +92,15 @@ defmodule PropCheck.Properties do
           %{module: module} = __ENV__
 
           module_default_opts = Module.get_attribute(module, :propcheck_default_opts) || [:quiet]
-          moduletag = Module.get_attribute(module, :moduletag) |> List.flatten()
-          describetag = Module.get_attribute(module, :describetag) |> List.flatten()
-          tag = Module.get_attribute(module, :tag) |> List.flatten()
+
+          # Allow using the Keyword module by filtering for tuple entries in the tags
+          filter = fn tags ->
+            Enum.filter(tags, &is_tuple/1)
+          end
+
+          moduletag = Module.get_attribute(module, :moduletag) |> List.flatten() |> filter.()
+          describetag = Module.get_attribute(module, :describetag) |> List.flatten() |> filter.()
+          tag = Module.get_attribute(module, :tag) |> List.flatten() |> filter.()
```

Comment:
Hah, using the new multi-line comment. 

Just to be nitpicky in stylist: I would recommend to add the `Enum.filter` directly: 
```elixir
moduletag = Module.get_attribute(module, :moduletag) |> List.flatten() |> Enum.filter(&is_tuple/1)
```

Reply:
Done and squashed into 7398745.